vhdbuilder/packer/windows-vhd-builder-sig.json

```diff
@@ -37,12 +36,15 @@
             "managed_image_resource_group_name": "{{user `resource_group_name`}}",
             "managed_image_name": "{{user `sig_image_name`}}",
             "shared_image_gallery_destination": {
-              "resource_group": "{{user `resource_group_name`}}",
-              "gallery_name": "{{user `sig_gallery_name`}}",
-              "image_name": "{{user `sig_image_name`}}",
-              "image_version": "{{user `sig_image_version`}}",
-              "replication_regions": ["{{user `location`}}"]
+                "resource_group": "{{user `resource_group_name`}}",
+                "gallery_name": "{{user `sig_gallery_name`}}",
+                "image_name": "{{user `sig_image_name`}}",
+                "image_version": "{{user `sig_image_version`}}",
+                "replication_regions": [
+                    "{{user `location`}}"
+                ]
             },
+            "polling_duration_timeout": "1h",
```

Comment:
@xuto2 @adilliadil  , I think we also need this change in polling_duration_timeout in the Linux side. I can also change Linux sig builder in the following commit or PR.

Reply:
Thanks! yes please help fix it.